lib/ansible/modules/packaging/language/pip.py

```diff
@@ -335,10 +335,11 @@ def _is_present(module, req, installed_pkgs, pkg_command):
     for pkg in installed_pkgs:
         if '==' in pkg:
             pkg_name, pkg_version = pkg.split('==')
+            pkg_name = Package.canonicalize_name(pkg_name)
```

Comment:
I'd probably move it to the initializer: https://github.com/ansible/ansible/issues/48095#issuecomment-436315372

Reply:
Make sense. Will do.

Reply:
Now I remember. It cannot be done in the initializer, because package name isn't split from the version there, so that means it would canonicalize the `name=version`, which isn't what we want. Anyway it's only needed here so it's fine that it stays.